tools/webpack-core/src/plugins/CopyExternalsPlugin.ts

```diff
@@ -24,6 +25,13 @@ class MissingExternalWarning extends WebpackError {
     Error.captureStackTrace(this, this.constructor);
   }
 }
+interface DependencyStruct {

```

Comment:
Not sure I like "struct" in the name here... how about naming this something like `PackageDependency` or even just `Dependency`?

Reply:
Changed